src/PhpWord/Shared/Html.php

```diff
@@ -957,6 +948,17 @@ protected static function parseImage($node, $element)
                 $tmpDir = Settings::getTempDir() . '/';
                 $match = [];
                 preg_match('/.+\.(\w+)$/', $src, $match);
+
+                if (empty($match) || !isset($match[1])) {
+                    $contentType = get_headers($src, 1)['Content-Type'];
+
+                    if (!array_key_exists($contentType, self::$contentTypeFileExtensionMap)) {
+                        throw new \Exception("Could not load image $src");
```

Comment:
Why do we throw here instead of adding an error text to the element like at the end of this function?

Reply:
Ah nice spot, I missed this one